### DISCUSSION TITLE
更改表情存储方式

### DISCUSSION THREAD

**Discussion by Lfhsheng** [walinejs/waline#741] [Q&A] [answered]
**问题描述 | Describe the bug**

由于jsdelivr出了点小问题，导致服务器迁移至香港，使得cdn在中国大陆变成了减速cdn

我觉得可以把表情上传到图床之类的地方

**问题网站 | Website URL**

https://waline-n41x.vercel.app/

**部署方式 | Deploy Type**

- 服务部署在哪里？ | Where your waline deploy?
  - [x] Vercel
  - [ ] Deta
  - [ ] CloudBase
  - [ ] Docker
  - [ ] VPS, Virtual Host
  - [ ] Deta
  - [ ] InspireCloud
  - [ ] Baidu CFC
  - [ ] Other
- 数据存储在哪里？| Where your comment data store?
  - [x] LeanCloud(https://leancloud.app)
  - [ ] LeanCloud(https://leancloud.cn)
  - [ ] CloudBase
  - [ ] MongoDB
  - [ ] MySQL
  - [ ] SQLite
  - [ ] PostgreSQL
  - [ ] GitHub
  - [ ] Deta Base
  - [ ] InspireCloud

**Comment by lizheming** [comment]
这个你可以自己下下来然后托管到其它的 OSS 上去吧

**Comment by Mister-Hope** [comment]
表情仓库是开源的，你可以直接克隆仓库内容自行托管

**Comment by Mister-Hope** [comment, marked as answer]
目前已经按照 `@waline/emoji` 发布了 npm 包，你可以自行找一个国内访问速度快的镜像。

**Reply by elkan1788** [reply]
@Mister-Hope  想请教一下如何制作纯文字的表情包。😁

**Reply by Mister-Hope** [reply]
暂不支持
